source/standards/practices/frontend-web-performance.html.md.erb

```diff
@@ -0,0 +1,110 @@
+---
+title: Frontend Web Performance
+last_reviewed_on: 2024-05-21
+review_in: 12 months
+weight: 3
+---
+
+# <%= current_page.data.title %>
+
+
+## Context
+
+By prioritizing frontend performance, governments can ensure that their online services are accessible, efficient, and user-friendly for all citizens, regardless of their device or connection. This fosters inclusivity, improves user satisfaction, and ultimately contributes to a more effective and trusted government. This aligns very closely with the topic of Frontend Performance in the [GDS Way](https://gds-way.digital.cabinet-office.gov.uk/standards/optimise-frontend-perf.html) and the [Service Manual](https://www.gov.uk/service-manual/technology/how-to-test-frontend-performance). Creating services that are performant is also a fundamental skill of being a [Frontend Developer](https://ddat-capability-framework.service.gov.uk/role/frontend-developer) in the Civil Service, according to the [Government Digital and Data Profession Capability Framework](https://ddat-capability-framework.service.gov.uk/).
+
+
+## Use a CDN
+
+Using a Content Delivery Network (CDN) is essential for optimal frontend performance. By mirroring website assets across its network, a CDN serves content from the geographically closest server, minimizing HTTP connection time and accelerating asset downloads. This is crucial, as a browser can only download assets after establishing an HTTP connection.
+
+CDNs are essential for fast page loads, as they minimize a user's Time to First Byte (TTFB).
+
+
+## Use Brotli compression
+
+Brotli, a lossless compression algorithm developed by Google, which offers up to 20% smaller file sizes than Gzip for web assets like CSS and JavaScript. Widely supported by browsers and CDNs [since late 2017](https://caniuse.com/brotli), Brotli seamlessly falls back to Gzip if unsupported, this browser decision is based on the `Accept-Encoding` and `Content-Encoding` headers during HTTP negotiation.
```

Comment:
I'm a bit wary of recommending this when its not currently turned on at the platform level, it could be but I don't think docs advocating it should be there before its possible for users of the platform to use it.

![image](https://github.com/hmcts/hmcts.github.io/assets/21194782/89dcc85b-cdd1-4065-94a9-d940b12359a2)

Reply:
I've included these comment as requested, thanks!